pkg/restapi/issuer/operation/models.go

```diff
@@ -46,6 +48,7 @@ type WalletConnect struct {
 
 // txnData contains session data.
 type txnData struct {
+	// Todo #580 rename IssuerID to ProfileID
```

Comment:
Why renaming issuerID to profileID???
#580 doesn't mention any of this.

Reply:
https://github.com/trustbloc/adapter/issues/580#issuecomment-1025900556 @sudeshrshetty  This was my reason,